projects/demo/src/app/player/player.component.ts

```diff
@@ -0,0 +1,38 @@
+import { Component, OnInit } from '@angular/core';
+import { Grange } from '../../../../grange/src';
+
+@Component({
+    selector: 'app-player',
+    templateUrl: 'player.component.html'
+})
+
```

Comment:
useless empty line (those are happening a lot when you create a component, is your IDE adding them by default? for me it's confusing to have an empty line between a decorator and the class it decorates)